src/Sylius/Bundle/UiBundle/Resources/assets/less/AdminLTE/skin-sylius.less

```diff
@@ -0,0 +1,71 @@
+@import "node_modules/bootstrap/less/mixins.less";
```

Comment:
It's ``assets``, not ``public`` - it won't be installed to web folder by Symfony. :)